### DISCUSSION TITLE
I think we should have a badge for Code review

### DISCUSSION THREAD

**Discussion by ptmy-it** [community/community#30569] [Other Feature Feedback, Questions, & Ideas]
I really love the badges and achievements system of GitHub. However, I think it is missing a badge for the **Code review** for this is also an essential activity on GitHub.

To be honest, I've learned a lot from code reviewing others' code and being code reviewed by others.

Since we are having the badge for the **Pull Request**. And well... the PRs should be reviewed and approved before merging. So I think it's fair to have one for the **Code review** as well. That makes sense, right?

**Comment by alexwholland** [comment]
Better then having the YOLO badge that awards merges without review.

**Comment by unixorn** [comment]
In my opinion, having someone else look at PRs is part of what makes Open Source so powerful.

**Comment by Trilochna** [comment]
_Yes i agree with you. I also feel that GitHub should provide more achievements and badges it is really fascinating and motivates to work._

**Comment by pappukumar35** [comment]
yes  its reality

**Comment by Adesoji1** [comment]
I have review 67 PR's last week,I didn't see any badge 😳😕

**Comment by pappukumar35** [comment]
profile inside

**Comment by sultanjulyan** [comment]
Badge for code review is a good one because need an effort for that, maybe bade for 10years member is also good.

**Comment by KnightChaser** [comment]
Wholeheartedly agree with you. Why GitHub still hasn't created that kind of badge?
